asapdiscovery-data/asapdiscovery/data/schema/ligand.py

```diff
@@ -617,6 +714,34 @@ def get_chemical_relationship(self, other: "Ligand") -> ChemicalRelationship:
             relationship |= ChemicalRelationship.DISTINCT
         return relationship
 
+    def sort_confs(self, by: str, ascending: bool = True) -> np.ndarray:
```

Comment:
This will be super useful, I would say make it clear that its by SD tag by making the name `sort_confs_by_sd_tag_value`